pkg/api/mlflow/service/artifact/validators.go

```diff
@@ -9,7 +9,7 @@ import (
 )
 
 // RelativePathRegExp checks for the sequence `./` or `../` in provided path.
-var RelativePathRegExp = regexp.MustCompile(`\.{1,2}\/`)
+var RelativePathRegExp = regexp.MustCompile(`\.{2}\/?`)
```

Comment:
this is not allowing valid paths like: `some..directory/`
the _only_ thing we should disallow is "directories that are named `..` anywhere in the path"

Reply:
fixed.